### DISCUSSION TITLE
SpotX and BTS Working (Guide or Resources)

### DISCUSSION THREAD

**Discussion by DakshKK** [SpotX-Official/SpotX#261] [Question]
I love tinkering. And recently came across SpotX and BTS.

As far as I understand the latter works on DLL injection, but unsure about the former.

I wanted to ask how they work, if further details could be given, and some ideas as to how was the Spotify app inspected to create the DLL, or whatever method SpotX is using.

**Comment by amd64fox** [comment]
SpotX blocks ads by changing the logic of the JS code.

Bts works with CEF binary files using hooks to block advertising requests directly through CEF
